controllers/reviews-controllers.js

```diff
@@ -37,3 +40,22 @@ exports.getReviewComments = (req, res, next) => {
       next(err);
     });
 };
+
+exports.postReviewComments = (req, res, next) => {
+  const { review_id } = req.params;
+  const newComment = req.body;
+
+  fetchReviewById(review_id)
```

Comment:
I dont know if you need to check to see if both the review and user exist here, you are probably more likely to get a foreign key violation from psql, this would mean that you wouldn't have to write all this blocking code, we currently wait for the review, then wait for the user, then wait for the post, it could be quite expensive and there is a lot more code added than relying on the feedback from the db :)